sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/channel/DistributeInfoTrackerTest.java

```diff
@@ -0,0 +1,91 @@
+package com.microsoft.appcenter.distribute.channel;
+
+import android.support.annotation.NonNull;
+
+import com.microsoft.appcenter.channel.Channel;
+import com.microsoft.appcenter.ingestion.models.AbstractLog;
+import com.microsoft.appcenter.ingestion.models.Log;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+
+import java.util.UUID;
+
+import static junit.framework.Assert.assertNull;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.mock;
+
+
+@SuppressWarnings("unused")
+@PrepareForTest({DistributeInfoTracker.class})
```

Comment:
You can remove curly braces when there is only 1, this what we do elsewhere, should use same code convention.

Reply:
Got it. Thank you.